packages/web/docs/src/components/cli-errors.tsx

```diff
@@ -0,0 +1,401 @@
+/* eslint-disable tailwindcss/no-custom-classname */
+import { ReactElement } from 'react';
+import type { GetStaticProps } from 'next';
+import { Callout, Code } from '@theguild/components';
+
+type CLIError = {
+  code: string;
+  title: string;
+  example: string;
+  exampleOutput: string;
+  fix: string;
+};
+
+export function ErrorDetails(props: CLIError): ReactElement {
+  return (
+    <>
+      <h3
+        id={`errors-${props.code}`}
+        className="_font-semibold _tracking-tight _text-slate-900 _mt-8 _text-2xl dark:_text-slate-100"
+      >
```

Comment:
These do not show up in the sidebar 🤔 
<img width="292" alt="image" src="https://github.com/user-attachments/assets/9256772d-5d9b-49f2-bac1-8228535d93b6" />